src/libraries/Microsoft.PowerFx.Interpreter/PowerFxInterpreterException.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.PowerFx.Interpreter
+{
+    [Serializable]
+    public class PowerFxInterpreterException : Exception
```

Comment:
Is the expectation that this be thrown from a derived table value?

Reply:
Absolutely